### PR TITLE
Fix registry stack create

### PR DESCRIPTION
**Pull request by jakolehm**

### CODE REVIEW

**Comment by SpComb** [comment]
## Testing
### Broken with 0.17.pre1

#### `kontena registry create`
```
 [error] {"stack"=>"Stack is required", "version"=>"Version is required", "registry"=>"Registry is required", "source"=>"Source is required"}
```

### Working with `fix/registry-stack-create`
#### `kontena registry create`
```
 [done] Deploying registry stack      

Docker Registry 2.2 is now running at registry.test.kontena.local.
Note: 
  - OpenVPN connection is needed to establish connection to this registry. See http://www.kontena.io/docs/using-kontena/vpn-access for details
  - you must set '--insecure-registry registry.test.kontena.local' to your client docker daemon before you are able to push to this registry
```

#### `kontena stack show registry`
```
registry:
  state: running
  created_at: 2016-11-21T14:55:02.641Z
  updated_at: 2016-11-21T14:55:02.641Z
  version: 0.16.3
  expose: api
  services:
    api:
      image: kontena/registry:2.2
      status: running
      revision: 1
      stateful: yes
      scaling: 1
      strategy: ha
      deploy_opts:
        min_health: 0.8
      dns: api.registry.test.kontena.local
```

#### `kontena service show registry/api`
```
test/registry/api:
  stack: test/registry
  status: running
  image: kontena/registry:2.2
  revision: 1
  stateful: yes
  scaling: 1
  strategy: ha
  deploy_opts:
    min_health: 0.8
  dns: api.registry.test.kontena.local
  env: 
    - REGISTRY_STORAGE_FILESYSTEM_ROOTDIRECTORY=/registry
    - REGISTRY_HTTP_ADDR=0.0.0.0:80
    - REGISTRY_HTTP_SECRET=81d145d4e5ba2b51607427f06dec88b3f500dcca56d40805
  net: bridge
  volumes:
    - /registry
  instances:
    registry-api-1:
      rev: 2016-11-21 14:55:03 UTC
      service_rev: 1
      node: snowy-night-25
      dns: api-1.registry.test.kontena.local
      ip: 10.81.128.3
      public ip: 95.85.4.70
      status: running
      exit code: 0
```

File: cli/lib/kontena/cli/registry/create_command.rb [hunk 1]
```diff
@@ -87,6 +87,11 @@ def execute
 
       data = {
         name: 'registry',
+        stack: 'kontena/registry',
+        version: Kontena::Cli::VERSION,
+        source: '---',
+        registry: 'file://',
```
Comment (SpComb) [inline cli/lib/kontena/cli/registry/create_command.rb]: Actually load the YAML from some CLI resource file?

Use something magical like `internal://registry`?

File an issue for moving these to the stack registry, so we don't forget?

**Comment by nevalla** [comment]
@jakolehm ready to merge?

**Comment by jakolehm** [comment]
@SpComb yes, I'll create a separate PR for the vpn.

@nevalla all done here, so merge ok.

**Pull request merged**